### PR TITLE
add clear variants doc

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [x] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/experiment/sdks/ios-sdk.md [hunk 1]
```diff
@@ -284,6 +284,22 @@ func all() -> [String:Variant]
...
+
+Clear all [variants](../general/data-model.md#variants) in the cache and storage.
+
+```swift
+func clear()
+```
+
+You can call `clear` after user logout to clear the variants in cach and storage.
+
+```java
```
Comment (qingzhuozhen) [inline docs/experiment/sdks/ios-sdk.md:297]: this should swift?

**Review by caseyamp — COMMENTED** [review]
Looks good to me, after one of your peers approves you're good to go on my end.

File: docs/experiment/sdks/ios-sdk.md (hunk 1 above)
Comment (yuhao900914) [reply docs/experiment/sdks/ios-sdk.md]: Typo, it has been fixed.

**Review by qingzhuozhen — APPROVED** [review]

File: docs/experiment/sdks/javascript-sdk.md [hunk 2]
```diff
@@ -286,6 +286,22 @@ all(): Variants
 
 ---
 
+### Clear
+
+Clear all [variants](../general/data-model.md#variants) in the cache and storage.
+
+```js
+clear() : void
```
Comment (bgiori) [inline docs/experiment/sdks/javascript-sdk.md:294]:
```suggestion
clear(): void
```

File: docs/experiment/sdks/react-native-sdk.md [hunk 3]
```diff
@@ -280,6 +280,22 @@ all(): Variants
 
 ---
 
+### Clear
+
+Clear all [variants](../general/data-model.md#variants) in the cache and storage.
+
+```js
+clear() : void
```
Comment (bgiori) [inline docs/experiment/sdks/react-native-sdk.md:288]:
```suggestion
clear(): void
```

**Review by bgiori — APPROVED** [review]

**Pull request merged**